### PR TITLE
SQL::selectFrom() pass columns as ref

### PR DESCRIPTION
**Pull request by fmanco**

### CODE REVIEW

File: osquery/sql/sql.cpp [hunk 1]
```diff
@@ -121,7 +121,7 @@ QueryData SQL::selectAllFrom(const std::string& table,
   return response;
 }
 
-QueryData SQL::selectFrom(const std::initializer_list<std::string> columns,
+QueryData SQL::selectFrom(const std::initializer_list<std::string>& columns,
```
Comment (akindyakov) [inline osquery/sql/sql.cpp]: To copy an `std::initializer_list` is very cheap. So, you can just put it here by value.
[proof](http://en.cppreference.com/w/cpp/utility/initializer_list). Quote: 
> Initializer lists may be implemented as a pair of pointers or pointer and length. Copying a std::initializer_list does not copy the underlying objects.

Comment (fmanco) [reply osquery/sql/sql.cpp]: Blame the linter 😄

Comment (fmanco) [reply osquery/sql/sql.cpp]: I suppose despite being cheap to copy passing by ref is not really a problem, correct?

Comment (akindyakov) [reply osquery/sql/sql.cpp]: Yes, sure

**Comment by akindyakov** [comment]
Looks good 👍

**Pull request merged**